site/content/docs/alerting-and-retries/retries.md

```diff
@@ -7,61 +7,107 @@ menu:
-This is mostly a stop-gap for not chalking up too many retries.
+On top of the time between attempts, you can also set: 
 
-{{<info >}}
-Note that the **maximum total retry duration** also includes the time it takes for a check to run. So if you set the maximum to 2 minutes,
-your check takes 1.5 minutes, you have 30 seconds left for retries.
-{{</info >}}
+* **maximum number of retries** - the maximum number of retries for this check or check group. 
+* **maximum total retry duration** - the maximum time a check can be in a retrying state. 
+This is a timeout to ensure the check finishes on a timely manner.
+    
+  {{<info >}}
+  Make sure to include the time your check needs to run when setting the **maximum total retry duration**. For example, if you set a maximum to 2 minutes, and your check takes 1.5 minutes, you have 30 seconds left for retries.
+  {{</info >}}
 
-### How often should I retry?
+#### How often should I retry?
 
-**For checks that run on lower frequencies**, e.g. once per hour, it makes sense to pick a strategy that retries more often,
-over longer stretches of time. Let's look at an example:
+**For checks that run on lower frequencies**, e.g. once per hour, it makes sense to pick a strategy that retries more often, over longer stretches of time. Let's look at an example:
 
 1. A check with a 1-hour frequency breaks at 13:00 CET.
 2. An exponential retry strategy with 3 retries and a 5-second interval is configured.
 3. This generates retries spaced apart by 5 seconds, 25 seconds, 125 seconds.
-4. Your check is now retried at about 13:00:05, 13:00:30 and 13:02:35. 
+4. Your check is now retried at about 13:00:05, 13:00:30 and 13:02:35.
 
-If in that ~3-minute period your app / system recovers, and this was just a fluke, you will not be alerted. If the issue persists,
-Checkly will alert you as per your alert settings.
+If in that ~3-minute period your app / system recovers, and this was just a fluke, you will not be alerted. If the issue persists, Checkly will alert you as per your alert settings.
 
-**For checks that run on higher frequencies**, e.g. once per minute, it makes sense to pick a strategy that retries less often,
-and spaced apart by shorter intervals.
+**For checks that run on higher frequencies**, e.g. once per minute, it makes sense to pick a strategy that retries less often, and spaced apart by shorter intervals.
 
 Of course, picking the right strategy depends on your use case, tolerance for intermittent failures, SLO levels and other factors.
 
-## Retry from the same region
+### Location-based check retries
 
-You can toggle if a check should be retried from the same location or not. 
-
-- If you enable this, a check that fails will be retried in the same location. 
-- If you disable this, a check that fails will be retried in a different location, selected from the location you selected for the check.
+You can decide if a check should be retried from the same location or not with the toggle: 
```

Comment:
Nitpick: The toggle is a checkbox - I had issues locating it initially, I wonder if it makes sense to clarify again that it can be found within the "retries & alerting" section - If you start reading at this section that might not be entirely clear

Reply:
Definitely, thanks for the catch, should be fixed now!